types/relay-compiler/tsconfig.json

```
@@ -1,11 +1,6 @@
 {
     "compilerOptions": {
         "module": "commonjs",
```

I think we still need a `lib` here.

```suggestion
        "module": "commonjs",
        "lib": [
          "es6"
        ],
```

https://github.com/DefinitelyTyped/DefinitelyTyped/runs/7890150582?check_suite_focus=true#step:6:86

```
1> relay-compiler failing:
1> Error: Must specify "lib", usually to `"lib": ["es6"]` or `"lib": ["es6", "dom"]`.
1>     at checkTsconfig (/home/runner/work/DefinitelyTyped/DefinitelyTyped/node_modules/@definitelytyped/dtslint/src/checks.ts:117:11)
1>     at testTypesVersion (/home/runner/work/DefinitelyTyped/DefinitelyTyped/node_modules/@definitelytyped/dtslint/src/index.ts:229:16)
1>     at runTests (/home/runner/work/DefinitelyTyped/DefinitelyTyped/node_modules/@definitelytyped/dtslint/src/index.ts:197:7)
```